SonarQube.Common/ConsoleLogger.cs

```diff
@@ -41,18 +45,28 @@ public ConsoleLogger(bool includeTimestamp)
         public void LogWarning(string message, params object[] args)
         {
             string finalMessage = this.GetFormattedMessage(Resources.Logger_WarningPrefix + message, args);
-            Console.WriteLine(finalMessage);
+
+            using (new ConsoleColorScope(WarningColor))
+            {
+                Console.WriteLine(finalMessage);
```

Comment:
I'll suppress these warnings - the purpose of the class is to log to the console.